geode-core/src/distributedTest/java/org/apache/geode/cache/client/internal/CacheServerSSLConnectionDUnitTest.java

```diff
@@ -105,7 +106,7 @@
   public static Collection<Boolean> data() {
     List<Boolean> result = new ArrayList<>();
     result.add(Boolean.TRUE);
-    result.add(Boolean.FALSE);
+    // result.add(Boolean.FALSE);
```

Comment:
I'll revert this - we need to test with both old-style SSL settings and new-style settings